### ISSUE TITLE
Add a Total Number of accounts

### ISSUE THREAD

**Issue by AngellitoX** [flagrow/user-directory#5]
It should be great to visually have:

- Total amount of registered accounts
- Possibly the amount of suspended acounts?
- An additional filter of Suspended accounts to keep track of who exactly are suspended.

**Comment by Luceos** [comment]
A filter of everyone suspended is brilliant 👍

**Comment by askvortsov1** [comment]
We'd need to add a suspended gambit, is that something we'd be ok with?

**Comment by clarkwinkelmann** [comment]
A suspended gambit should be added to the suspend extension IMO.

**Comment by askvortsov1** [comment]
I agree with that, but that wouldn't take effect until the beta 14 release, if not later. How would y'all feel about PRing both gambits to the respective repos, and having a temporary copy here until that's released?

**Comment by clarkwinkelmann** [comment]
> but that wouldn't take effect until the beta 14 release

Unless someone is in urgent need of that feature, I would prefer we get the feature in Flarum core and only then release it here. With the new Flarum release cycles and the fact most of us are part of the core team, I think we should have no issue getting this shipped with the coming release.

That's less maintenance required, and it might take a few weeks to code or review any PR anyway.

**Comment by clarkwinkelmann** [comment]
Gambit was added in https://github.com/flarum/suspend/pull/23 and is available in beta 14.
